packages/react-core/src/components/DualListSelector/examples/DualListSelectorComposable.tsx

```diff
@@ -141,21 +145,36 @@ export const DualListSelectorComposable: React.FunctionComponent = () => {
         } options selected`}
         searchInput={buildSearchInput(true)}
         actions={[buildSort(true)]}
+        listHeight="270px"
       >
-        <DualListSelectorList>
-          {availableOptions.map((option, index) =>
-            option.isVisible ? (
-              <DualListSelectorListItem
-                key={index}
-                isSelected={option.selected}
-                id={`composable-available-option-${index}`}
-                onOptionSelect={e => onOptionSelect(e, index, false)}
-              >
-                {option.text}
-              </DualListSelectorListItem>
-            ) : null
-          )}
-        </DualListSelectorList>
+        {availableFilter !== '' && availableOptions.filter(option => option.isVisible).length === 0 && (
+          <EmptyState variant={EmptyStateVariant.small}>
+            <EmptyStateIcon icon={SearchIcon} />
+            <Title headingLevel="h4" size="md">
+              No results found
+            </Title>
+            <EmptyStateBody>No results match the filter criteria. Clear all filters and try again.</EmptyStateBody>
+            <Button variant="link" onClick={() => onFilterChange('', true)}>
+              Clear all filters
+            </Button>
+          </EmptyState>
```

Comment:
Nit, not a blocker: we could take this and have it returned from a `build...` function similar to the buildSort and buildSearchInput ones since both panes are using the same empty state minus the `onFilterChange` args. That could possibly be a followup at some point if we wanted to have a buildPane function similar to the composable tree example.